wagtail/tests/testapp/models.py

```diff
@@ -454,3 +457,18 @@ class MTIChildPage(MTIBasePage):
 class AbstractPage(Page):
     class Meta:
         abstract = True
+
+
+@register_setting
+class TestSetting(BaseSetting):
+    title = models.CharField(max_length=100)
+    email = models.EmailField()
```

Comment:
The ``max_length`` needs to be explicitly set on this field, to fix the Django 1.7 migration issue.